integration/js/script/run-test

```diff
@@ -77,67 +77,119 @@ cd ../
 . ~/.bashrc
 
 case $1 in
-   audio)
-      r audio_test.config.json
-      r app_quit_audio_test.config.json
-      r meeting_leave_audio_test.config.json
-      ;;
-   video)
-      r video_test.config.json
-      r video_test_processor.config.json
-      #  TODO: Re-enable simulcast test after it has been fixed
-      # r video_test_simulcast.config.json
-      r app_quit_video_test.config.json
-      r meeting_leave_video_test.config.json
-      r reconnection_test.config.json
-      ;;
-   video_test_app)
-      r test_app_video_test.config.json
-      r test_app_video_pin_test.config.json
-      ;;
-   meeting_end)
-      r meeting_end_test.config.json
-      ;;
-   content_share_test_suite_one)
-      r content_share_screen_capture_test.config.json
-      r meeting_leave_content_share_test.config.json
-      r content_share_video_test.config.json
-      ;;
-   content_share_test_suite_two)
-      r content_share_only_allow_two_test.config.json
-      r content_share_join_later_test.config.json
-      r app_quit_content_share_test.config.json
-      ;;
-   data_message)
-      r data_message_test.config.json
-      ;;
-   meeting_readiness_checker)
-      r meeting_readiness_checker_test.config.json
-      ;;
-   messaging)
-      r messaging_session_test.config.json
-      ;;
-   transcription)
-      r transcription_test.config.json
-      ;;
-   background_blur)
-      r background_blur_test.config.json
-      ;;
-   echo_reduction)
-      r echo_reduction_test.config.json
+   integration)
+   case $2 in
+      audio)
+         r configs/integrationTest/audio_test.config.json
```

Comment:
Upto you to if you want to fix nothing major, but would be great to have a variable something like below:
```
testDir = 'configs/integrationTest'
..
...
..
r $testDir/audio_test.config.json
```
Same for "configs/browserCompatibilityTest".